src/streams/mod.rs

```diff
@@ -441,15 +484,109 @@ impl Streams {
         }
     }
 
-    pub(crate) fn change_stream_rx_ssrc(&mut self, ssrc_from: Ssrc, ssrc_to: Ssrc) {
+    pub(crate) fn change_stream_rx_ssrc(
+        &mut self,
+        ssrc_from: Ssrc,
+        ssrc_to: Ssrc,
+        rtx_to: Option<Ssrc>,
+    ) {
```

Comment:
I prefer having a separate function for changing RTX, this overloads the purpose a bit too much.

Reply:
Aight, we have a quite a lot of methods that take an optional `rtx: Option<Ssrc>` so I tried to mirror that